hack/test.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+#
+# This script runs tests and generates a report file.
+set -e
+echo "" > coverage.txt
+for d in $(go list ./... | grep -v '/vendor/' | grep -v 'integration_test'); do
```

Comment:
I think it would be better to assign the listing of go files to a new variable and then use it in the for loop .

Reply:
done.